### ISSUE TITLE
Still developing?

### ISSUE THREAD

**Issue by thepixture** [nickberens360/atomic-docs#64]
Great tool!

I would like to know if the project will be further developed. Last update was a year ago.

**Comment by nickberens360** [comment]
@thepixture yes we are! Currently working on the next version. Not sure yet when It will be ready to go yet though.
Thanks, Nick

**Comment by LucPestille** [comment]
Any update or rough ETA? The current master branch is so close to being perfect for my needs, and from what I can see of v4 it solves lots of my wants...

**Comment by nickberens360** [comment]
@LucPestille so the back end dev on our team is too busy with client so the project has stalled. I'm really just a designer and have taken this version as far as I can. My next step is to reach out to the dev community and see who can help. I'll keep you updated. I think it's a valuable project and worth the time!

**Comment by malithmcr** [comment]
Hey @nickberens360, 
I could help with some frontend and backend stuff. please let me know any way I could be helpful.
